src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/masonry-decorator/masonry-view.js

```diff
@@ -447,7 +448,13 @@ define([
              */
             itemLoadedHandler: function($item) {
                 this.sandbox.masonry.refresh('#' + this.masonryGridId, true);
-                this.sandbox.dom.removeClass($item, constants.loadingClass);
+                $item.removeClass(constants.loadingClass);
+                // lock the height of the image container
+                $item.find('.masonry-image').height($item.find('.masonry-image').height());
```

Comment:
When does this happen? Why has that to be done? Is it because the masonry grid is reordered all the time otherwise?